src/SmoFacade/User.cs

```diff
@@ -1,18 +1,62 @@
 ﻿namespace AgDatabaseMove.SmoFacade
 {
+  using System.Collections.Generic;
+  using Smo = Microsoft.SqlServer.Management.Smo;
+
+  public class UserProperties
+  {
+    public string Name { get; set; }
+    public IEnumerable<RoleProperties> Roles { get; set; }
+    public Smo.DatabasePermissionSet Permissions { get; set; }
+    public string LoginName { get; set; }
+  }

```

Comment:
what's the diff between `LoginName` and `Name`?

Reply:
`LoginName` is referring to the name of the associated login, `Name` refers to the name of the user.